train.py

```diff
@@ -91,13 +98,39 @@ def main(opts):
             Dict({"name": "crop", "ignore": False, "height": 32, "width": 32})
         ]
 
+    # ------------------------------------------
+    # -----  Check if auto adventv2 works  -----
+    # ------------------------------------------
+    is_auto_adventv2 = opts.train.lambdas.advent.is_auto_adventv2
+    if is_auto_adventv2:
+        assert opts.tasks == [
+            "m"
+        ], "Auto adventv2 only works if mask is the only task to be trained!"
+        opts.train.epochs = opts.train.lambdas.advent.stage_one_epochs
+        if opts.train.resume:
+            if opts.train.lambdas.advent.stage_one_epochs == 0:
+                print("Continue on stage two training")
+            else:
+                print("Continue on stage one training")
+
     # -------------------
     # -----  Train  -----
     # -------------------
     trainer = Trainer(opts, comet_exp=exp)
     trainer.logger.time.start_time = time()
     trainer.setup()
-    trainer.train()
+    # start training if the expected training epochs is not 0
+    if opts.train.epochs != 0:
+        trainer.train()
+    if is_auto_adventv2:
```

Comment:
I think there needs to be a check around the number of epochs because the default is `100000000` meaning endless phase 1...